csrankings-c.csv

```diff
@@ -537,6 +537,7 @@ Chengjun Liu,NJIT,http://cs.njit.edu/liu,TvBcLGMAAAAJ
 Chengkai Li 0001,University of Texas at Arlington,http://ranger.uta.edu/~cli,v8ZQDf8AAAAJ
 Chenglin Miao,Iowa State University,https://clmiao.github.io,QFs2tZoAAAAJ
 Chenglu Wen,Xiamen University,https://asc.xmu.edu.cn/t/wenchenglu,JOoZUmUAAAAJ
+Chengming Zhang 0006,University of Houston,https://www.uh.edu/nsm/computer-science/people/faculty/,F4TtJZEAAAAJ

```

Comment:
Replace with actual home page: https://chengmingzh8.github.io/

Reply:
Same for all home pages. They should not be a generic departmental directory page.